chromedriver_autoinstaller/utils.py

```diff
@@ -225,6 +225,14 @@ def get_matched_chromedriver_version(chrome_version, no_ssl=False):
         for good_version in good_version_list["versions"]:
             if good_version["version"] == chrome_version:
                 return chrome_version
+        # get last major version
+        matched_chromedriver_version = None
+        for good_version in good_version_list["versions"]:
+            if get_major_version(good_version["version"]) == get_major_version(chrome_version):
+                matched_chromedriver_version = good_version["version"]
+            if get_major_version(good_version["version"]) > get_major_version(chrome_version):
+                break
+        return matched_chromedriver_version
```

Comment:
It's probably reasonable to assume Google will give us the versions in sequential order, so at this risk of over engineering this we could compare revision numbers like this:
```suggestion
        matched_chromedriver_version = None
        major_version_matches = {}
        for good_version in good_version_list["versions"]:
            if get_major_version(good_version["version"]) == get_major_version(chrome_version):
                major_version_matches[good_version["revision"]] = good_version["version"]
            if get_major_version(good_version["version"]) > get_major_version(chrome_version):
                break
        if len(major_version_matches):
            matched_chromedriver_version = major_version_matches[max(major_version_matches)]
        return matched_chromedriver_version
```